modules/plugin/jdbc/jdbc-oracle/src/test/java/org/geotools/data/oracle/OracleDataStoreAPIOnlineTest.java

```diff
@@ -56,4 +56,21 @@ public void testSequenceDetection() throws IOException {
         SequencedPrimaryKeyColumn seqPk = (SequencedPrimaryKeyColumn) pk;
         assertEquals("ROAD_FID_SEQ", seqPk.getSequenceName());
     }
+
+    /**
+     * Test if tables that should be hidden are absent in the list of type names.
+     *
+     * @throws IOException if any
+     * @see org.geotools.data.oracle.OracleDialect#includeTable(String, String, java.sql.Connection)
+     */
+    public void testHiddenTables() throws IOException {
+    	String[] typenames = dataStore.getTypeNames();
+    	for (String name:typenames) {
+    		System.out.println(name);
```

Comment:
No system outs in tests please

Reply:
:cry: sorry about that, will see to it asap